examples/connect/connect-with-sidecar-sssd.yaml

```diff
@@ -0,0 +1,116 @@
+# WARNING: no shared storage, so no persistence for this system
+sharedStorage:
+  enabled: false
+
+pod:
+  volumes:
+    - name: sssd-emptydir
+      emptyDir: {}
+    - name: sssd-config
+      configMap:
+        name: sssd-config
+        defaultMode: 0600
+    - name: pam-config
+      configMap:
+        name: pam-config
+        defaultMode: 0755
+  volumeMounts:
+    - name: sssd-emptydir
+      mountPath: /var/lib/sss
+    - name: pam-config
+      mountPath: /etc/pam.d/common-account
+      subPath: common-account
+    - name: pam-config
+      mountPath: /etc/pam.d/common-session
+      subPath: common-session
+    - name: pam-config
+      mountPath: /etc/pam.d/common-auth
+      subPath: common-auth
+    - name: pam-config
+      mountPath: /etc/pam.d/common-password
+      subPath: common-password
+  sidecar:
+    - name: sssd
+      image: "some-sssd-image"
+      imagePullPolicy: "IfNotPresent"
+      securityContext: {}
+      volumeMounts:
+        - name: sssd-emptydir
+          mountPath: /var/lib/sss
+        - name: sssd-config
+          mountPath: /etc/sssd/
```

Comment:
We might be able to use this pattern to support `runAsCurrent` user with off-host execution. I don't think it would require any changes to Connect since we already assume uid/gid/passwd consistency between Connect and the content Pod.

Reply:
😮 That's pretty cool!

Reply:
@aronatkins pointed out that rsandbox/PAM password forwarding might pose a problem here, but it's worth a spike to see how far this gets us and if there are any other limitations.